.github/workflows/build-deb-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Test Deb Package
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
```

Comment:
It's better to have this build on push to the default branch instead of using `workflow_dispatch`